airbyte-integrations/connectors/source-linnworks/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.7"
+name = "source-linnworks"
+description = "Source implementation for Linnworks."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/linnworks"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
+include = "source_linnworks"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0.61.2"
+vcrpy = "^6.0.0"
+pendulum = "==2.1.2"
```

Comment:
I believe `pendulum` is included as a part of `airbyte-cdk`, should it be removed?

Reply:
Didn't realize that, thanks for the heads up! Verified that it comes included with the CDK and updated the poetry/lock file